Client/ClientRoom.cpp

```diff
@@ -370,22 +480,31 @@ void ClientRoom::_initSharedResources()
     _initSpritesForEntities();
 }
 
-void ClientRoom::_initSystems()
+void ClientRoom::_initSystems(bool isSolo)
```

Comment:
Why did you not use an intern variable for Solomode ?

Reply:
Because we need this info into 1 function, not all the Client Room Class